app/models/request.rb

```diff
@@ -27,4 +28,32 @@ def items_hash
       [key, Item.order("created_at ASC").find_by(partner_key: key)]
     end.to_h
   end
+
+  def family_request_reply
+    {
+      "organization_id": organization_id,
+      "partner_id": partner_id,
+      "requested_items": item_requests.map do |ir|
+        {
+          "item_id": ir.item_id,
+          "count": ir.quantity
+        }
+      end
+    }
+  end
+
+  def self.parse_family_request(family_request)
+    request = Request.new(organization_id: family_request['organization_id'], partner_id: family_request['partner_id'])
+    request.item_requests = []
+
+    family_request['requested_items'].each do |item|
+      item_request = ItemRequest.new
+      item_request.item = Item.find(item['item_id'])
+
+      item_request.quantity = item_request.item.default_quantity * item['person_count']
+      request.item_requests << item_request
+    end
+
+    request
```

Comment:
Note to future us: 

We don't yet have enough information, but I think if we have one or two more additional request "flavors", we should implement STI on this.